docs/contributing.md

```diff
@@ -0,0 +1,22 @@
+Contribution Guide
+==================
+
+Licensing
+---------
+
+This project is licensed using the [BSD-3-Clause license](https://github.com/projectatomic/osbs-client/blob/master/LICENSE). When submitting pull requests please make sure your commit messages include a signed-off-by line. You can do this by using `git commit --signoff`.
```

Comment:
Maybe a link to this is helpful? https://help.github.com/articles/signing-commits-using-gpg/